app/assets/javascripts/controllers/live_metrics/ad_hoc_metrics_controller.js

```diff
@@ -149,6 +149,7 @@ ManageIQ.angular.app.controller('adHocMetricsController', ['$http', '$window', '
 
     dash.refreshTenant = function() {
       initialization();
+      httpUtils.getMetricTags();
```

Comment:
Right 👍 , actually I think the initial bug was that it was removed from `initialization` and not from `refreshTenant` , thanks, it will look nicer in `initialization`.